R/t-distribution/app-sam.R

```diff
@@ -74,22 +75,25 @@ ui <- miniPage(
           class = "align-items-center",
           sliderInput(inputId = "df",
                       label = "Degrees of Freedom", min = 1, max = 200, value = 1, step = 1)
-                     
-          # numericInput(inputId = "df", label = "Degrees of Freedom", STARTING_DF, 
+
+          # numericInput(inputId = "df", label = "Degrees of Freedom", STARTING_DF,
           #              min = 1, max = 200, value = 1, step = 1
           #              )
           ),
+         div(# critical values readout
```

Comment:
added critical values readout back in - how this should look/feel/act is still up for debate